lib/libutee/amble/Apache License.txt

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
I don't think that is necessary since we have the following comment in our LICENSE file:
`Unless it has its own copyright/license embedded in its body, each source file
is subject to the following license terms:`